tealiumlibrary/src/main/java/com/tealium/core/consent/TealiumConfigConsentManager.kt

```diff
@@ -37,4 +41,26 @@ var TealiumConfig.consentManagerPolicy: ConsentPolicy?
         value?.let {
             options[CONSENT_MANAGER_POLICY] = it
         }
+    }
+
+/**
+ * Sets the consent expiration.
+ */
+var TealiumConfig.consentExpiry: ConsentExpiry?
+    get() = options[CONSENT_EXPIRY] as? ConsentExpiry
+    set(value) {
+        value?.let {
+            options[CONSENT_EXPIRY] = it
+        }
+    }
+
+/**
+ * Sets an optional callback to be triggered upon consent expiration.
+ */
+var TealiumConfig.onConsentExpiration: (()->Unit)?
```

Comment:
@tamayok @christinasund 
I'm not convinced we even need a separate listener for this - since we already have one (below) for consent updates that will already notify when the Status is reverted to `UNKNOWN` 
```
interface UserConsentPreferencesUpdatedListener: Listener {
    fun onUserConsentPreferencesUpdated(userConsentPreferences: UserConsentPreferences, policy: ConsentManagementPolicy)
}
```